irteus/Makefile

```diff
@@ -30,6 +30,18 @@ INSTALLBINDIR=$(IRTEUSDIR)/$(ARCHDIR)/bin
 INSTALLOBJDIR=$(IRTEUSDIR)/$(ARCHDIR)/obj
 INSTALLLIBDIR=$(IRTEUSDIR)/$(ARCHDIR)/lib
 
+# check bullet version
+BULLET_VER_MAJOR:=$(shell pkg-config bullet --modversion | cut -f1 -d.)
+BULLET_VER_MINOR:=$(shell pkg-config bullet --modversion | cut -f2 -d.)
+export BULLET_GE_2_83=$(shell [ $(BULLET_VER_MAJOR) -gt 2 -o \( $(BULLET_VER_MAJOR) -eq 2 -a $(BULLET_VER_MINOR) -ge 83 \) ] && echo true)
```

Comment:
I do not understand exactly but if we do not have bullet it outputs too many `1: [: -gt: unexpected operator`
```
HOGE=$(shell echo "ng")

all:
        echo "ok"
        echo $(HOGE)
        echo $(HOGE)
        echo $(HOGE)
```

Reply:
Fixed in https://github.com/euslisp/jskeus/pull/538/commits/9a6aeabae976816935a14e45d17ee43bd99b762f. referenced https://stackoverflow.com/a/5586785